### ISSUE TITLE
Will Samsung devices loose their warranty?

### ISSUE THREAD

**Issue by Hibiki0506** [VegaBobo/DSU-Sideloader#88]
Hello, I wanted to ask if Samsung devices will loose their warranty by installing any GSIs? 
Because unlocking the bootloader will not trip the knox counter on devices like the Galaxy S23U.

Thanks in advance.

**Comment by VegaBobo** [comment]
That's a good question, in one of my videos comments, a person said that it won't trip, but i cannot give you any trust if  this behavior really happens.

I think it should trip, since it likely to trigger AVB, but i cannot test to give you a concrete answer, i own only one Samsung device in which the Knox is already tripped.

I'll let this issue open, if someone want to give it a try, keep in mind to just test it if you don't mind tripping Knox.

**Comment by Fede2782** [comment]
Yes, because you have to replace vbmeta which makes Knox 0x1 in order to boot gsi.
